cdhweb/urls.py

```diff
@@ -78,24 +74,10 @@
     # may complicate testing with pa11y-ci
     url(r"^sitemap\.xml$", sitemap, {'sitemaps': sitemaps}, name='sitemap'),
 
-    # wagtail paths
-    # NOTE temporarily make wagtail pages available at pages/ so that they can
-    # coexist with mezzanine urls
-    url(r'^cms/', include(wagtailadmin_urls)),
-    url(r'^documents/', include(wagtaildocs_urls)),
-    url(r'^pages/', include(wagtail_urls)),
-
     # let mezzanine handle everything else
     url("^", include("mezzanine.urls")),
-]
 
-if settings.DEBUG:
-    # serve user-uploaded media in debug mode
-    urlpatterns += [
-        url(r'^media/(?P<path>.*)$', serve, {
-            'document_root': settings.MEDIA_ROOT,
-        }),
-    ]
+]
```

Comment:
Don't we still need a version of this? Looks like we can update to newer syntax https://docs.djangoproject.com/en/3.1/howto/static-files/#serving-files-uploaded-by-a-user-during-development